.github/dockerfile/Dockerfile.debian

```diff
@@ -12,12 +12,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG GO_VERSION=1.18
-FROM golang:${GO_VERSION} AS builder
+FROM debian:stretch
+
+ARG GO_VERSION=1.18.5
+
+RUN apt update \
+    && apt install -y zip upx pkg-config libczmq-dev build-essential debhelper jq zip
+
+RUN case $(arch) in \
```

Comment:
In debian/ubuntu, you can use `$(dpkg --print-architecture)` instead of `$(arch)`